angular.json

```diff
@@ -115,5 +115,8 @@
       }
     }
   },
-  "defaultProject": "teammates"
-}
+  "defaultProject": "teammates",
+  "cli": {
+    "analytics": "5b44a57d-2fa6-4bda-8654-8d8d27cc0df1"
```

Comment:
Where is this from/why is this needed?

Reply:
Must have added it by mistake, I will remove it